changelog/unreleased/fix-users-by-claim-lookup-binary-uuid.md

```diff
@@ -0,0 +1,9 @@
+Bugfix: GetUserByClaim fixed for Active Directory
+
+The reva ldap backend for the users and groups service didn't not
+hex escape binary uuids in LDAP filter correctly this could cause
+problems in Active Directory setups for services using the
+GetUserByClaim CS3 request with claim "userid".
```

Comment:
```suggestion
The reva ldap backend for the users and groups service did not
hex escape binary uuids in LDAP filter correctly this could cause
problems in Active Directory setups for services using the
GetUserByClaim CS3 request with claim "userid".
```
double not?